lib/webauthn/attestation_statement/base.rb

```diff
@@ -5,6 +5,13 @@ module AttestationStatement
     class Base
       class NotSupportedError < StandardError; end
 
+      ATTESTATION_TYPE_NONE = "none"
+      ATTESTATION_TYPE_BASIC = "basic"
+      ATTESTATION_TYPE_SELF = "self"
+      ATTESTATION_TYPE_ATTCA = "attca"
+      ATTESTATION_TYPE_ECDAA = "ecdaa"
+      ATTESTATION_TYPE_BASIC_OR_ATTCA = "basic_or_attca"
```

Comment:
W3C Recommendation seem to use first letter uppercase, e.g. `None` instead of `none` or `AttCA` instead of `attca`.

Reply:
Also, can we move these out of `Base` namespace?